src/plugins/data/common/osd_field_types/types.ts

```diff
@@ -93,3 +93,27 @@ export enum OSD_FIELD_TYPES {
   NESTED = 'nested',
   HISTOGRAM = 'histogram',
 }
+
+/** @public **/
+export enum OPENSEARCH_SQL_TYPES {
```

Comment:
nit: for consistency `OPENSEARCH_SQL_FIELD_TYPES`

Reply:
updated